rules.gradle

```diff
@@ -24,11 +24,11 @@ dependencies {
 }
```

Comment:
These changes were made in your previous CL. You might need to merge `google:master` into your branch so it becomes up to date and doesn't show this diff. This page might be useful do do that: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

Reply:
Oh yeah, I was confused why those were still there. Thank you!